### PR TITLE
site(commitlog): link href to commit hash on GitHub

### PR DESCRIPTION
**Pull request by RickyC0626**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -682,7 +682,7 @@
     #commit_log {
       display: grid;
       grid-template-columns: 1fr;
-      gap: 24px;
```
Comment (CanadaHonk) [inline index.html:685]: Unrelated?

**Review by CanadaHonk — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (RickyC0626) [reply index.html:685]: I thought it looked cleaner with less gap. What do you think?

**Review by CanadaHonk — APPROVED** [review]
Thanks!
